.github/workflows/release.yaml

```
@@ -0,0 +1,35 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches:
+      - main
```

I think we can test it out on this branch with the following:

```suggestion
    branches:
      - main
      - owensmallwood/add-gh-action-to-build-and-push-docker-image
```

---

ah, good call 👍